src/nanorc/cfgsvr.py

```diff
@@ -79,7 +98,82 @@ def save_on_resume(self, data: dict) -> dict:
         f.write(json.dumps(data, indent=2))
         f.close()
 
-
+    
+    def save_on_stop(self, run:int):
+        pass
+        
+
+        
+
+class DBConfigSaver:
+    def __init__(self, installation_id:str, socket:str):
+        self.API_SOCKET = socket
+        auth = credentials.get_login("runregistrydb")
+        self.API_USER = auth.user
+        self.API_PSWD = auth.password
+        self.timeout = 2
+        self.installation_id = installation_id
```

Comment:
Dis we say `installation` or was it `apparatus` at the end?

Reply:
\facepalm. That should be apparatus.